extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -229,12 +243,23 @@ export class AmpLightboxViewer extends AMP.BaseElement {
 
   /**
    * Toggle description box if it has text content
+   * @param {boolean=} opt_display
    * @private
    */
-  toggleDescriptionBox_() {
+  toggleDescriptionBox_(opt_display) {
     this.updateDescriptionBox_();
+    dev().assert(this.descriptionBox_);
+    if (opt_display == undefined) {
+      opt_display = this.descriptionBox_.classList.contains('hide');
+    }
     if (this.descriptionBox_.textContent) {
-      this.descriptionBox_.classList.toggle('hide');
+      if (opt_display) {
```

Comment:
can be simplified to `this.descriptionBox_.classList.toggle('hide', opt_display);`

Reply:
I found that IE doesn't support `classList.toggle` with second argument (see [this](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)), so we'd better use the dumb way.

Reply:
we polyfill it https://github.com/ampproject/amphtml/pull/6400 and use it a lot actually.

Reply:
done